packages/build-tools/cmd/posix_helpers.go

```diff
@@ -114,18 +114,18 @@ var rmCmd = &cobra.Command{
 
 		for _, item := range items {
 			info, err := os.Stat(item)
-			if err != nil && !force {
-				return eris.Wrapf(err, "Could not stat %s", item)
-			}
-
-			if info.IsDir() && !recursive {
+			if err != nil {
+				if !(force && eris.Is(err, os.ErrNotExist)) {
+					return eris.Wrapf(err, "Could not stat %s", item)
+				}
+			} else if info.IsDir() && !recursive {
 				return eris.Errorf("%s is a directory but -r wasn't passed", item)
 			}
 		}
 
 		for _, item := range items {
 			err := os.RemoveAll(item)
-			if err != nil && (!force || !eris.Is(err, os.ErrNotExist)) {
+			if err != nil {
```

Comment:
You want to keep this check since missing files aren't removed from `items` during the first iteration and thus are still present during this iteration.

Reply:
`RemoveAll` actually does not throw `os.ErrNotExist` as it is essentially `rm -f` in syscall form. So there is no need to handle this down below.